flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java

```diff
@@ -161,7 +161,10 @@ private static void syncDatabase(
             jobName =
                     String.format(
                             "%s-Doris Sync Database: %s",
-                            type, config.getString("database-name", "db"));
+                            type,
+                            "mongodb".equals(type)
+                                    ? config.getString("database", "db")
+                                    : config.getString("database-name", "db"));
```

Comment:
It is better to unify it as database-name. You can change the config in MongoDBDatabaseSync.buildCdcSource to read

Reply:
thanks for you reviewing, another commit changed it.